paasta_tools/kubernetes_tools.py

```diff
@@ -1005,6 +1005,12 @@ def get_desired_instances(self) -> int:
             )
         return instances
 
+    def get_annotations(self, kube_client: KubeClient) -> Dict:
```

Comment:
I don't think this function is really necessary anymore, it's literally just wrapping `get_annotations_for_deployment`.